test_codecept/e2e/features/app/registerOtherOrgNavigation.feature

```diff
@@ -183,6 +183,7 @@ Feature: Register other org, Navigations
             | Email address                                               | Provide your contact details                               |
             | What regulators are you (as an individual) registered with? | What regulator are you (as an individual) registered with? |
 
+    @ignore
```

Comment:
Do we understand why this is failing and plan to resolve?

Reply:
This issue as a result of the issue with the address lookup added by the feature team.

Reply:
Code has been updated now.